creating-virtual-machines/virtio-win.adoc

```diff
@@ -1,53 +1,186 @@
 Windows driver disk usage
 -------------------------
 
-*Provides*:
+Purpose of this document is to explain how to install virtio drivers for Microsoft Windows
+running in a fully virtualized guest.
 
-* virtio drivers
-* QEMU Guest Agent
+Do I need virtio drivers?
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. Without the virtio drivers, you cannot use
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_getting_started_guide/sec-virtualization_getting_started-products-virtualized-hardware-devices#sec-Virtualization_Getting_Started-Products-paravirtdevices[paravirtualized] hardware properly.
+It would either not work, or will have a severe performance penalty.
+
+For more details on configuring your guest please refer to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_deployment_and_administration_guide/chap-guest_virtual_machine_device_configuration[Guest Virtual Machine Device Configuration].
+
+
+Which drivers I need to install?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are usually up to 8 possible devices that are required to run Windows
+smoothly in a virtualized environment:
+
+* Balloon, the balloon driver, applies to PCI Device in the Other devices group
+* vioserial, the paravirtual serial driver, applies to PCI Simple Communications Controller in the Other devices group.
+* NetKVM, the network driver, applies to Ethernet Controller in the Other devices group. Available only if a virtio NIC is configured.
+* viostor, the block driver, applies to SCSI Controller in the Other devices group.
+* viocsci, the SCSI block driver, applies to SCSI Controller in the Other devices group.
+* viorng, the entropy source driver, applies to PCI Device in the Other devices group.
+* qemupciserial, the emulated PCI serial driver, applies to PCI Serial Port in the Other devices group.
+* qxl, the paravirtual video driver, applied to Microsoft Basic Display Adapter in the Display adapters group.
+* pvpanic, the paravirtual panic driver, applies to Unknown device in the Other devices group.
+
+[NOTE]
+====
+Some drivers are required in the installation phase. Namely, when installing Windows
+on virtio storage device, requires you to provide an appropriate driver.
```

Comment:
please elaborate. It's pretty logical, but this is a user documentation, so some practical obvious examples would be appreciated